src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/actions/CreateSpecializedTypeAction.java

```diff
@@ -48,7 +48,7 @@ public static final boolean createSpecializedType(final StructuralFeature redefi
             Application.getInstance().getGUILog().log(parent.getQualifiedName() + " is not editable. Skipping creating specialization.");
             return true;
         }
-        if (redefinedAttribute.getType() instanceof Classifier && !(redefinedAttribute.getType() instanceof Property)) {
+        if (!(redefinedAttribute.getType() instanceof DataType) && !(redefinedAttribute.getType() instanceof Property)) {
```

Comment:
Why did we make this change?

Reply:
Realized that DataType evaluates to true under instanceof Classifier test.